examples/literate/D_simulatingFloatingPlatforms.jl

```diff
@@ -0,0 +1,394 @@
+        [0.0, 0.0], #ac
+        [0.0, 0.0], #twist_d
+        [0.0, 0.0], #rhoA
+        [1e18, 1e18], #EIyy
+        [1e18, 1e18], #EIzz
+        [1e18, 1e18], #GJ
+        [1e18, 1e18], #EA
+        [0.0, 0.0], #rhoIyy
+        [0.0, 0.0], #rhoIzz
+        [0.0, 0.0], #rhoJ
+        [-8.6588, 0.0], #zcm
+        [0.0, 0.0], #ycm,
+        [0.0, 0.0], #a
+        [0.0, 0.0], #EIyz
+        [0.0, 0.0], #alpha1
+        [0.0, 0.0], #alpha2
+        [0.0, 0.0], #alpha3
+        [0.0, 0.0], #alpha4
+        [0.0, 0.0], #alpha5
+        [0.0, 0.0], #alpha6
+        [0.0, 0.0], #rhoIyz
+        [0.0, 0.0], #b
+        [0.0, 0.0], #a0
+        [0.0, 0.0], #aeroCenterOffset
+    )
+end
+
+nothing
+
+# We also want to provide some additional general inputs to enable floating simulation.
+# `hydroOn` being the principal and obvious one, but also the input files needed by
+# HydroDyn and MoorDyn, which are the external libraries that calculate the hydrodynamic
+# and mooring loads to send to the bottom side mesh. These input files include:
+#  - interpOrder: the degree of interpolation used to predict the future states within
+#    HydroDyn and MoorDyn, used in the `HD_UpdateStates` and `MD_UpdateStates` functions
+#  - hd_input_file: the base HydroDyn input file with a `.dat` extension
+#  - md_input_file: the base MoorDyn input file with a `.dat` extension
+#  - ss_input_file: the sea state input file with a `.dat` extension used to define the environmental conditions of the sea.
+#    This is used within HydroDyn.
+#  - potfilefile: the prefix of the potential flow files.
+#    At minimum, the directory must contain the .1, .3, and .hst WAMIT output files.
+#    This is also used within HydroDyn.
+# See the OpenFAST documentation for HydroDyn (https://openfast.readthedocs.io/en/main/source/user/hydrodyn/input_files.html)
+# and MoorDyn (https://moordyn.readthedocs.io/en/latest/inputs.html) for more
+# information about how to format these input files. For simplicity here, we will
+# use predefined input files for the OC4 semisubmersible platform, which comes with
+# OpenFAST and is copied to the `data` folder here.
+
+if AModel=="AD"
+    AD15On = true
+else
+    AD15On = false
+end
+hydroOn = true
+interpOrder = 2
+hd_input_file = "data/HydroDyn.dat"
+md_input_file = "data/MoorDyn.dat"
+ss_input_file = "data/SeaState.dat"
+potflowfile = "data/potflowdata/marin_semi"
+
+inputs = OWENS.Inputs(;analysisType = structuralModel,
+tocp = [0.0,100000.1],
+Omegaocp = [RPM,RPM] ./ 60,
+tocp_Vinf = [0.0,100000.1],
+Vinfocp = [Vinf,Vinf],
+numTS,
+delta_t,
+AD15On,
+aeroLoadsOn = 2,
+hydroOn,
+interpOrder,
+hd_input_file,
+md_input_file,
+ss_input_file,
+potflowfile)
+
+nothing
+
+# We also will need to specify where the HydroDyn and MoorDyn libraries exist on the computer.
+# These will differ based on operating system and system setup, so change these as needed.
+hd_lib = "$path/../../bin/HydroDyn_c_binding_x64"
```

Comment:
need to change these to nothing so it will use the auto installed libraries in the ci/cd environment